src/index.js

```diff
@@ -127,6 +128,7 @@ const HANDLERS = {
   'guidance:high-page-views-low-form-nav': highPageViewsLowFormNavGuidance,
   'guidance:high-page-views-low-form-views': highPageViewsLowFormViewsGuidance,
   'geo-brand-presence': geoBrandPresence,
```

Comment:
not part of this PR, but @davidaurelio @irenelagno why do we have `geo-brand-presence` and also `detect: geo-brand-presence`? can we get rid of the former?

Reply:
detect:… is where mystique sends the sheets when done with geo-brand-presence